cluster/images/conformance/go-runner/main.go

```diff
@@ -23,11 +23,15 @@ import (
 	"os"
 	"os/signal"
 	"path/filepath"
+	"strings"
 
 	"github.com/pkg/errors"
 )
 
 func main() {
+	if strings.Contains(os.Args[0], "run_e2e.sh") || strings.Contains(os.Args[0], "gorunner") {
+		log.Print("warn: calling test with e2e.test is deprecated, please rely on container manifest to invoke executable")
```

Comment:
maybe we should set some rough deadline for when it will no longer be supported and tell the user?
thenw e can also stop storing it with this name.

Reply:
i thought about that but didn't feel like the right person to make such call 😅 do you think 1.25 is reasonable?

Reply:
okay I submitted 1.25 for now, let me know if we should change that 😄